wiki/content/graphql/lambda/overview.md

```diff
@@ -0,0 +1,195 @@
++++
+title = "Overview"
+weight = 1
+[menu.main]
+    parent = "lambda"
+    identifier = "lambda-resolvers-overview"
++++
+
+Lambda provides a way to write your custom logic in JavaScript, integrate it with your GraphQL schema, and execute it using the GraphQL API in a few easy steps:
+
+- Setup a Dgraph cluster with a working lambda server (not required for [Slash GraphQL](https://dgraph.io/slash-graphql) users)
+- Declare lambda queries, mutations, and fields in your GraphQL schema as needed
+- Define lambda resolvers for them in a JavaScript file
+
+This also simplifies the job of UI developers, allowing them to work in JavaScript, without knowing any other backend language.
+
+Dgraph doesn't execute your custom logic itself. It makes external HTTP requests to a user-defined lambda server. That means, you can deploy your lambda logic into the same Kubernetes cluster as your Dgraph instance. 
```

Comment:
Drop the second sentence and just say that "Slash will do this for you".

Reply:
Done